app/views/admin/configs/show.html.erb

```diff
@@ -234,11 +224,59 @@
                                              placeholder: Constants::SiteConfig::DETAILS[:recaptcha_secret_key][:placeholder] %>
                           </div>
                         </div>
+                        <div class="mt-6">
+                          <% if SiteConfig.allow_both_email_signup_and_login %>
+                            <button
+                              class="crayons-btn crayons-btn--danger"
+                              data-action="click->config#activateEmailAuthModal"
+                              <%= SiteConfig.invite_only_mode ? 'disabled' : ''%>>
+                              <% if SiteConfig.invite_only_mode %>
+                                <span
+                                  class="crayons"
+                                  data-tooltip="<%= Constants::SiteConfig::DETAILS[:invite_only_mode][:cannot_disable_email] %>">
+                                  Disable
+                                <span>
+                              <% else %>
+                                Disable
+                              <% end %>
+                            </button>
+                          <% end %>
+                          <% if SiteConfig.allow_both_email_signup_and_login %>
+                            <button
+                              class="crayons-btn crayons-btn--secondary"
+                              data-action="click->config#hideEmailAuthSettings">
+                              Close
+                            </button>
+                          <% else %>
+                            <button
+                              class="crayons-btn crayons-btn--secondary"
+                              data-action="click->config#disableEmailAuth">
+                              Undo
+                            </button>
+                          <% end %>
+                        </div>
                       </fieldset>
                     </div>
                   </div>
                 </section>
 
+                <div class="form-group">
```

Comment:
I moved the 3rd-party Authentication Dropdown below the Email Authentication block, to place it closer to the associated key fields @joshpuetz